### DISCUSSION TITLE
Requests for Write Access to the Repo

### DISCUSSION THREAD

**Discussion by sharon-digipres** [Digital-Preservation-Coalition/PythonStudyGroups#48] [Announcements]
Post here to request write access to the PSG Repo. This will allow you to post code to [your relevant study group folder](https://github.com/Digital-Preservation-Coalition/PythonStudyGroups/tree/main/2024%20Cohort%202).

**Comment by barsness13** [comment]
Hi!  I'd love write access -- I'd like to upload some things I worked on in Cohort 1 (Group 13) as well as the new cohort (Group 15).

**Reply by mutanthumb** [reply]
Hi Sarah-
I just sent you an invite to join as an outside collaborator.  If you have any issues let us know.

Thanks,
susn

**Comment by evacmueller** [comment]
Hello, I'd also like access to the Repo. I'm from Group 17

**Reply by mutanthumb** [reply]
Hi Eva-
I just sent you an invite, let me know if you have any issues.
Thanks,
susan

**Comment by DeanAyotte** [comment]
Hi, I'd like write access to the repo. I'm in group 12.

Thanks!

Dean

**Reply by mutanthumb** [reply]
Hi Dean-
I just sent you an invite. I also approved your pull request. 
-susan

**Comment by aadams-nmrn** [comment]
Hello can I have access to Group 11 please. Thanks Amy.

**Reply by mutanthumb** [reply]
I just sent you an invite, let me know if you have any issues. -susan

**Comment by ccllaa** [comment]
Hi Sharon
I'd love to have write access to the repository of group 14. (Sorry, I previously made an issue instead of a comment).

**Reply by mutanthumb** [reply]
Hi I just sent you an invite, please let me know if you have any issues! 
Thanks, susan

**Comment by laradfs** [comment]
Could I have write access please? I'm in group 14.

**Reply by mutanthumb** [reply]
Hey Lara-
This is Susan from UMich BTAA, I just sent you an invite let me know if you have any issues.
